action.yml

```diff
@@ -53,8 +53,13 @@ inputs:
   fix:
     description: 'Run with --fix? Defaults to false'
     required: false
+  npmInstall:
+    description: 'Run npm install (or yarn) for you. If you do not use this option, do the npm install step yourself, before running this action. Defaults to true'
```

Comment:
```suggestion
    description: 'Force run npm ci (or yarn) for you. If you do not use this option, be sure to install the project dependencies before running this action.  By default it will run if a node_modules directory is not found or this is set to true'
```

Reply:
Perfect 👍